### PR TITLE
[Docs] Added Sinhalese translation

### PR DESCRIPTION
**Pull request by Ln11211**
Hello,
This PR is related to issue https://github.com/AndrGab/medirepoWeb/issues/76 . Sinhalese is the language of Sri Lanka , I have noticed that the translation is missing, so I have added the Sinhalese translation, hope this helps :)

### CODE REVIEW

**Comment by Ln11211** [comment]
hey @AndrGab , do you like it? :)

**Comment by Ln11211** [comment]
hi @AndrGab , I fixed a typo in Telugu file, it's just a small one, and apart form that, can you please review and merge this PR?

File: src/i18n.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ import contentZhCN from './locales/zh_CN/content.json';
 import contentMa from './locales/ma/content.json';
 import contentVi from './locales/vi/content.json';
 import contentSl from './locales/sl/content.json';
+import contentSl from './locales/si/content.json';
```
Comment (AndrGab) [inline src/i18n.js]:
```suggestion
import contentSi from './locales/si/content.json';
```

File: src/locales/si/si.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (AndrGab) [inline src/locales/si/si.json]: the filename has to be "content.json"

**Review by AndrGab — CHANGES_REQUESTED** [review]

**Review by AndrGab — APPROVED** [review]

**Pull request merged**